changelog.d/16304.doc

```diff
@@ -0,0 +1 @@
+Add alpine-linux package as official install option
```

Comment:
```suggestion
Link to the Alpine Linux community package for Synapse.
```

(Sorry, I'd do make this change myself but it seems that maintainers aren't granted edit rights to your PR.)

Reply:
Well its a github issue for forks in orgs ...

Reply:
Not sure I follow---IIRC there is a [tickbox option for this when opening a fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).

In any case, it's all good now. Thanks for bearing with me!

Reply:
If the fork is in an organization there's no tickbox: https://github.com/orgs/community/discussions/5634